### PR TITLE
Remove the bazel test runner option

### PR DESCRIPTION
**Pull request by DaveShuckerow**
Remove it and clear out code paths that involve the user preference.

### CODE REVIEW

File: src/io/flutter/run/bazelTest/FlutterBazelTestConfigurationEditorForm.java [hunk 1]
```diff
@@ -77,7 +73,7 @@ public void customize(final JList list,
     });
     // Only enable scope changes if the new bazel test runner is enabled.
     // If the new runner is disabled, all scopes will be blaze target-level.
```
Comment (devoncarew) [inline src/io/flutter/run/bazelTest/FlutterBazelTestConfigurationEditorForm.java]: We can update this comment.

Comment (DaveShuckerow) [reply src/io/flutter/run/bazelTest/FlutterBazelTestConfigurationEditorForm.java]: Done.

**Comment by devoncarew** [comment]
ping re: landing this

**Comment by DaveShuckerow** [comment]
I started a thread on the g3 users announcement list for this.

Policy is to allow 1 week for discussion before landing the potentially breaking change.

**Comment by devoncarew** [comment]
We are starting the next build. If we want this is this month's release, we'll need to land this.

**Comment by DaveShuckerow** [comment]
No objections internally, landing it.

**Pull request merged**